### PR TITLE
Touches up some styles on 1995

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1349

---
> Space above branch of service label should match other fields

image

---
> On the review page, the page edit button is touching the bottom border in Firefox

image

---
> The line spacing in a multi line label doesn't match the space between the label and a text field/area.  This makes fields like our education objective textarea look a little weird. We should even that spacing up.

The change I made affects the spacing of non-multi-line labels as well, but only for labels before tel / email / text `input`s, `textarea`s, and `select` boxes:

image

---
> Accordion styling on confirmation page should match normal accordion styles

image

image

---
> Capitalize "back" in button on confirmation page

image

### CODE REVIEW

File: src/sass/_normalize.scss
Comment (cvalarida) [inline src/sass/_normalize.scss]: This didn't really seem to fit anywhere else, so I put it in a normalize scss file. It should go in uswds probably, but I don't know how to go about doing that, so for now, it's in here.

File: src/js/edu-benefits/1995/containers/ConfirmationPage.jsx
Comment (cvalarida) [inline src/js/edu-benefits/1995/containers/ConfirmationPage.jsx]: This section is copied from `src/js/common/schemaform/review/ReviewCollapsibleChapter.jsx`. That particular component isn't able to be used from the confirmation page, but I can pull this chunk out into a common component if desired.

File: src/sass/_normalize.scss
Comment (jbalboni) [inline src/sass/_normalize.scss]: I think this is ok, but you should check the other uses of `<dl>` to make sure it looks ok (looks like there are only a few).

File: src/js/edu-benefits/1995/containers/ConfirmationPage.jsx
Comment (jbalboni) [inline src/js/edu-benefits/1995/containers/ConfirmationPage.jsx]: I'd like to keep the `form-review-panel` styles from leaking onto this page. You should be able to do an accordion with just the uswds styles (https://standards.usa.gov/components/accordions/)

File: src/sass/modules/_m-form-elements.scss
Comment (jbalboni) [inline src/sass/modules/_m-form-elements.scss]: Assuming Alex signs off on how this looks, I think we will just want to change this for the same inputs that the uswds styles set a top margin for.

**Review by jbalboni — COMMENTED** [review]
Made a couple comments, but looks alright to me. Alex should take a look at the input changes and make sure they're ok.

**Comment by jbalboni** [comment]
@goldenmeanie Can you look at the input spacing changes?

**Comment by goldenmeanie** [comment]
Really nice improvement overall!

Date input looks like it partially escaped

screen_shot_2017-02-24_at_2_35_09_pm

**Comment by cvalarida** [comment]
Removed `.form-review-panel` from the confirmation accordion:

File: src/sass/_normalize.scss
Comment (cvalarida) [inline src/sass/_normalize.scss]: The only other place I saw a `<dl>` was in the [Rx GlossaryList](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/rx/components/GlossaryList.jsx#L25). It should be fine since it's just normalizing. Takes what Chrome was already doing and making sure other browsers do the same.

File: src/sass/modules/_m-form-elements.scss
Comment (jbalboni) [inline src/sass/modules/_m-form-elements.scss]: We should do the above and also move these styles to the edu-benefits styles, since this would affect all pages on vets.gov. I think that's what we want to do eventually, but Alex needs to run it by other people first.

**Review by jbalboni — APPROVED** [review]
Looks good, can be merged after my comment has been addressed.

**Pull request merged**
